src/engine/strat_engine/pool.rs

```diff
@@ -360,12 +359,12 @@ impl StratPool {
 
 impl Pool for StratPool {
     fn create_filesystems<'a, 'b>(&'a mut self,
-                                  specs: &[&'b str])
+                                  specs: &[(&'b str, Option<Sectors>)])
```

Comment:
You'd like a header comment added (not updated), right?  This ok?
    /// Create a filesystem in the Pool.
    /// If the same name is passed multiple times, the size associated with
    /// the last item is used.